src/status_im/utils/universal_links/core.cljs

```diff
@@ -76,7 +77,24 @@
 (rf/defn handle-community
   [cofx {:keys [community-id]}]
   (log/info "universal-links: handling community" community-id)
-  (navigation/navigate-to-cofx cofx :community {:community-id community-id}))
+  (navigation/navigate-to-cofx cofx :community {:community-id community-id})
+)
+
+(rf/defn handle-navigation-to-desktop-community-from-mobile
+  {:events [:handle-navigation-to-desktop-community-from-mobile]}
+  [{:keys [db]} cofx deserialized-key]
+  (navigation/navigate-to-cofx cofx :community {:community-id deserialized-key})
+)
+
+(rf/defn handle-desktop-community
+  [cofx {:keys [community-id]}]
+  (status/deserialize-and-compress-key
+   community-id
+   (fn [deserialized-key]
+     (rf/dispatch [:handle-navigation-to-desktop-community-from-mobile cofx (str deserialized-key)])
+   )
```

Comment:
There are some dangling parentheses in this file.

Reply:
I wish these were caught with z-print or somethin